Paws4Me/Paws4Me/Display All Adoptable Pets/View Controllers/AllPetDetailViewController.swift

```diff
@@ -34,6 +35,15 @@ class AllPetDetailViewController: UIViewController {
     }
 
     // MARK: - IBActions
+    @IBAction private func logOutTappedButton(_ sender: UIButton) {
+        do {
+            try Auth.auth().signOut()
```

Comment:
Code like this does not belong in the `ViewController` 😄

Reply:
I agree. I will remove it to my reppository